tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -78,366 +71,358 @@ describe('Home - Page Content', () => {
+      // Test that JOIN button starts enabled
+      cy.contains('button.v-btn', 'Play').should('not.be.disabled');
+      // Sign up 2 users and subscribe them to game
+      cy.signupOpponent(opponentOne);
+      cy.subscribeOpponent(gameData.gameId);
+      cy.signupOpponent(opponentTwo);
+      cy.subscribeOpponent(gameData.gameId);
 
-    it('Re-enable join when a user leaves a full lobby', () => {
-      /**
-       * Set up:
-       * Create game, sign up two other users, subscribe them to the game, leave one user
-       */
-      cy.createGamePlayer({ gameName: 'Test Game', isRanked: false }).then((gameData) => {
-        // Test that JOIN button starts enabled
-        cy.contains('[data-cy-join-game]', 'Play').should('not.be.disabled');
-        // Sign up 2 users and subscribe them to game
-        cy.signupOpponent('secondUser@aol.com', 'myNewPassword');
-        cy.subscribeOpponent(gameData.gameId);
-        cy.signupOpponent('thirdUser@facebook.com', 'anotherUserPw');
-        cy.subscribeOpponent(gameData.gameId);
-
-        // Test that join button is now disabled
-        cy.contains('[data-cy-join-game]', 'Play').should('be.disabled');
-
-        cy.leaveLobbyOpponent(gameData.gameId);
-        cy.contains('[data-cy-join-game]', 'Play').should('not.be.disabled');
-      });
-    });
-  });
+      // Test that join button is now disabled
+      cy.contains('[data-cy-join-game]', 'Play').should('be.disabled');
 
-  describe('Spectating games', () => {
-    it('Spectates a game', () => {
-      cy.createGamePlayer({ gameName: 'Test Game', isRanked: false }).then(({ gameId }) => {
-        // Test that JOIN button starts enabled
-        cy.contains(`[data-cy-join-game=${gameId}]`, 'Play').should('not.be.disabled');
-        // Sign up 2 users and subscribe them to game
-        cy.signupOpponent(playerOne.username, playerOne.password);
-        cy.subscribeOpponent(gameId);
-        // Opponents start game, it appears as spectatable
-        cy.readyOpponent(gameId);
-        cy.signupOpponent(playerTwo.username, playerTwo.password);
-        cy.subscribeOpponent(gameId, 1);
-        cy.contains(`[data-cy-join-game=${gameId}]`, 'Play').should('be.disabled');
-
-        // Switch to spectate tab
-        cy.get('[data-cy-game-list-selector=spectate]').click();
-        cy.get('[data-cy=no-spectate-game-text]').should('contain', 'No Games Available to Spectate');
-
-        // The other game starts -- should now appear in spectate list
-        cy.readyOpponent(gameId);
-        cy.get(`[data-cy-spectate-game=${gameId}]`).click();
-
-        cy.url().should('include', '/spectate/');
-        cy.window()
-          .its('cuttle.app.config.globalProperties.$store.state.game')
-          .then((gameState) => {
-            expect(gameState.id).to.not.eq(null);
-          });
-      });
+      cy.leaveLobbyOpponent(gameData.gameId);
+      cy.contains('[data-cy-join-game]', 'Play').should('not.be.disabled');
     });
+  });
+});
```

Comment:
These were previously wrapped in a `describe('Open Games')` block. Let's re-add that wrapping layer around these tests